code/week5/bunny.h

```diff
@@ -0,0 +1,28 @@
+#pragma once
+
+#include <iostream>
+
+#include "prey.h"
+
+// subclass of Prey through that a subclass of Animal
+class Bunny : public Prey {
+	static int meter_maid_number;
+	int required_tickets;
+	int ticket_num;
+public:
+	// constructor
+	Bunny(std::string name, int required_tickets);
+
+	// overwrites the do_something method from animal and will be called
+	// even if a Bunny is referenced as a Animal
+	// override annotation has the compiler check that the method in overriden
+	// correctly i.e. this prototype matches one in Animal of the same name.
+	virtual void do_something() override;
```

Comment:
The virtual keyword is redundant and generally omitted in the presence of the override keyword.